packages/weather/index.ts

```diff
@@ -1,18 +1,14 @@
-import { gql } from 'apollo-server';
 import weatherData from './weather.json';
-import { buildServer } from 'lib';
-import * as console from 'console';
-
-const typeDefs = gql`
-    type Weather @key(fields: "code") {
-        code: ID
-        name: String
-    }
+import { buildServer } from '../lib';
```

Comment:
My IDE is telling me to shorten this to just `'lib'`. I'm guessing it's because of the workspace set up, it can view it as a package.

Reply:
Strange, mine didn't do that at all. I just did this to get it compiling. Did you do anything specific in your ide to setup the workspaces?

Reply:
no - I didn't touch it, it seems to see the `package.json` files from the other packages and use the package names as if they were node dependencies.

Reply:
it works this way too - it just gives me a syntax suggestion to shorten it

Reply:
<img width="828" alt="Screenshot 2021-09-15 at 11 51 17" src="https://user-images.githubusercontent.com/23258009/133420603-dba108e1-5406-42a1-a077-a4d75a2c02ed.png">